.devcontainer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/devcontainers/python:0-3.10
+
+COPY requirements.txt /tmp/
+RUN chmod +x /tmp/requirements.txt
```

Comment:
This isn't a script - why does it need to be executable?

Reply:
Good catch!